gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -194,10 +194,18 @@ class QueryFormatter(object):
   """
 
   class _QueryMacros(enum.Enum):
+    COUNT_AA_INFO_FIELD_QUERY = 'SELECT COUNT(AA) AS cnt FROM {TABLE_NAME}'
```

Comment:
Please use the queries inline in each test. These macros are meant to only be used for queries that are common across ~all tests. See [here](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/testing/integration/presubmit_tests/medium_tests/test_1000_copies_of_valid_4_2_with_merge.json#L24) for an example of inlining queries.

Reply:
Done!